extensions/algolia/fieldgroups/algolia-event.example.1.json

```
@@ -0,0 +1,21 @@
+{
+  "algolia-event": {
```

```suggestion
  "algoliaEvent": {
```
After making the changes suggested for `algolia-event.schema.json`, the example will change as specified above.